src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -1213,7 +1213,7 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
       }
     } else {
       const liveEdge = this._getLiveEdge();
-      if (this._liveEdge !== liveEdge) {
```

Comment:
better to change the way we are calculating the segment duration and calculate it only once according to the manifest like it does in shake player for example 
but this is a refactor task for the next sprint
iv open an issue see [FEC-12438](https://kaltura.atlassian.net/browse/FEC-12438)